### PR TITLE
Remove env variables from json config files

### LINKED ISSUE #23: Replace production .env file by newly added lambda environment variables
See https://aws.amazon.com/blogs/aws/new-for-aws-lambda-environment-variables-and-serverless-application-model, no need for a custom production env or code anymore!

### PR DESCRIPTION
**Pull request by ashkan18**
This PR removes all environment variables set currently in `.env.json` and `project.json` to switch to Lambda environment variable setting.

Fixes #23 

# Migration

- [x] Update Travis CI setting to include these deleted env variables
- [x] Update production lambda and add same env variables there

### CODE REVIEW

**Comment by ashkan18** [comment]
Builds are failing on PR builds because of:
> Travis CI makes encrypted variables and data available only to pull requests coming from the same repository. These are considered trustworthy, as only members with write access to the repository can send them.
Pull requests sent from forked repositories do not have access to encrypted variables or data.

from [here](https://docs.travis-ci.com/user/pull-requests). We need to find a better way for setting test env variables, maybe use staging for tests?

**Comment by dblock** [comment]
When you add an ENV variable to Travis, you can check the "show in logs" option, which will make it appear in pull requests. Obviously don't put anything really secret there, but these test keys are fine. Once you've added them, restart the build for this PR and feel free to merge.

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ This should return a welcome message.
 
 You need Artsy Alexa `project.production.json` and access to the Artsy AWS infrastructure. If you work at Artsy, you will find it in 1Password in the Engineering vault under `elderfield (Alexa) project.production.json`.
 
+Make sure proper environment variables are set in AWS Lambda settings.
```
Comment (dblock) [inline DEPLOYMENT.md]: I would say which ones, especially ENV.

**Comment by ashkan18** [comment]
All right, this should be ready, I also made `ARTSY_BASE_API_URL` an env and I point to staging in Travis CI. Let me know if you rather to have one commit for all these changes.

File: functions/artsy/models/api.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ var superagent = require('superagent')
 
 function Api(xappToken) {
     this.xappToken = xappToken;
+    this.base_url = process.env["ARTSY_BASE_API_URL"];
```
Comment (dblock) [inline functions/artsy/models/api.js]: I would just default this to api.artsy.net

**Pull request merged**